react/IOMessageWithMarkers.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useMemo } from 'react'
+import { useCssHandles } from 'vtex.css-handles'
+
+import IOMessage from './IOMessage'
+import { IOMessageWithMarkers as IOMessageWithMarkersType } from './typings/IOMessageWithMarkers'
+
+const IOMessageWithMarkers: IOMessageWithMarkersType = ({
+  label,
+  markers = [],
+  componentName,
+  values = {},
+}) => {
+  const CSS_HANDLES = useMemo(() => markers.map(marker => {
+    return `${componentName}_${marker}`
```

Comment:
Did we discuss this handle naming pattern? Should we be using `_` as separators?

Reply:
That's a great question